Source/Fuse.Controls.Native/iOS/TextEdit.uno

```diff
@@ -652,18 +652,7 @@ namespace Fuse.Controls.Native.iOS
 
 		TextInputActionStyle ITextEdit.ActionStyle
 		{
-			set
-			{
-				switch(value)
-				{
-					case TextInputActionStyle.Default: SetActionStyle(Handle, extern<int>"UIReturnKeyDefault"); break;
-					case TextInputActionStyle.Done: SetActionStyle(Handle, extern<int>"UIReturnKeyDone"); break;
-					case TextInputActionStyle.Next: SetActionStyle(Handle, extern<int>"UIReturnKeyNext"); break;
-					case TextInputActionStyle.Go: SetActionStyle(Handle, extern<int>"UIReturnKeyGo"); break;
-					case TextInputActionStyle.Search: SetActionStyle(Handle, extern<int>"UIReturnKeySearch"); break;
-					case TextInputActionStyle.Send: SetActionStyle(Handle, extern<int>"UIReturnKeySend"); break;
-				}
-			}
+			set { /* Does not apply to MultilineTextEdit */ }
 		}
```

Comment:
Keeping this feels a bit silly, as a setter which does nothing is kinda unexpected. Can't we remove this from the ITextEdit, and just check if the ITextEdit is `SingleLineTextEdit`, and call the method directly instead?

Reply:
Have to keep the setter around due to legacy reasons. This will be fixed when I get around to split the `ITextEdit` interface into `ITextInput` and `ITextView`. But that must be done in a separate PR as it would involve a lot more code changes